phys2bids/phys2bids.py

```diff
@@ -191,7 +191,7 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
     # Check options to make them internally coherent pt. II
     # #!# This can probably be done while parsing?
     indir = utils.check_input_dir(indir)
-    if chtrig < 1:
+    if chtrig < 0:
         raise Exception('Wrong trigger channel. Channel indexing starts with 1!')
```

Comment:
This message has to be updated accordingly (starts with 1, or it's zero for automatic search)